apps/labs/posts/automated-accessibility-testing-with-bokeh.md

```diff
@@ -0,0 +1,307 @@
+---
+title: 'Automated Accessibility Testing with Bokeh'
+authors: [chiemezuo-akujobi]
+published: September 30, 2024
+description: 'Data is only useful to the extent that people can access it.
+  Let me walk you through one of several steps toward making Bokeh data
+  visualizations more accessible.'
```

Comment:
Rewrite this please so that somebody knows that this blog post is like a summary of a three month internship

```suggestion
description: 'Data is only useful to the extent that people can access it.
  Let me walk you through one of several steps toward making Bokeh data
  visualizations more accessible.'
```